### ISSUE TITLE
Search EU -> not EU -> EU

### ISSUE THREAD

**Issue by ole-tange** [ixmaps/website2017#83]
In theory the system supports searching: from EU via not EU to EU.

But it returns very few routes and they are clearly wrong.

**Comment by colinmccann** [comment]
Yes, we have been very focused on Canada and have very few EU routes. This is partially because we have not had many EU traceroute submitters, but also because we don't yet have a clear data set for EU router geolocation (a very large job, see the faq section 'How does IXmaps locate routers geographically and map them).

**Comment by ole-tange** [comment]
Are you aware that you can get VPN services at < 100 USD that will give you exit points in most countries world wide? This should be enough to get a start in EU.

**Comment by ole-tange** [comment]
The link in the Faq is down: "How IXmaps Geolocates Routers."

"We also use latency calculations to help locate routers, or at least spot where Maxmind gives impossible results."

I am puzzled that I find quite a few that should have been caught by this. E.g, 2 ms from DK to UK.

**Comment by Andrew-Clement** [comment]
There are indeed many mis-located routers, including as you point out some that are inconsistent with SoL calcs. So far we only use latency when manually correcting suspiciously located routers. We would like to automate latency calculations to flag routes for correction and where possible to re-locate them near trustworthy landmarks, but found it difficult to do reliably. If you have an interest in this we'd welcome your thoughts.  Given the variations in latency, latency becomes increasingly dodgy along a route, so we figure we can only do this close to origin points that have low jitter in the local networks. 

Yes, we are aware of cheap VPNs, but we prefer to promote a crowd sourcing approach to traceroute generation - to acheive a variety of origin locatoins and ISPs. With the recent launch in the past week we have added >200K routes, many form outside north america.  This puts us in a much better position to do this location analysis.

**Comment by ole-tange** [comment]
Instead of putting a router at a point I would put in an area: Given the speed of light and the delay for each hop we can often infer an area where the router must be. E.g. a router in UK can never be 2 ms from Denmark, where as a router in Sweden could be.

Every router that we are unsure of should be in the TR00-set, so everyone helps narrowing the area where the router can be, as the router must be in the intersection of the areas determined from the different traces.
